presto-native-execution/presto_cpp/main/PrestoExchangeSource.cpp

```diff
@@ -370,7 +370,12 @@ void PrestoExchangeSource::getMemoryUsage(
     int64_t& currentBytes,
```

Comment:
Should we remove this parameter if it is not used and change the name to getPeakMemoryUsage() ?